enabler/index.html

```diff
@@ -1,89 +1,204 @@
+        !"%VITE_CTP_CLIENT_SECRET%".startsWith("%") &&
+        "%VITE_CTP_CLIENT_SECRET%";
+      var __VITE_CTP_PROJECT_KEY__ =
+        !"%VITE_CTP_PROJECT_KEY%".startsWith("%") && "%VITE_CTP_PROJECT_KEY%";
+      var __VITE_PROCESSOR_URL__ =
+        !"%VITE_PROCESSOR_URL%".startsWith("%") && "%VITE_PROCESSOR_URL%";
     </script>
     <script src="../dev-utils/session.js"></script>
   </head>
   <body>
-
-
     <main role="main">
-
       <section class="jumbotron text-center">
         <div class="container">
           <form action="/action_page.php">
             <div class="text-center mb-4">
               <h1 class="h3 mb-3 font-weight-normal">Dev Site</h1>
               <p>Use this site for development purposes</p>
             </div>
-            <div class="form-group">
-              <label for="cartId">Cart ID:</label>
-              <input type="text" class="form-control" id="cartId">
+            <div class="row">
+              <div class="form-group col-md-6">
+                <div>
+                  <label for="cartId">Cart ID:</label>
+                  <input type="text" class="form-control" id="cartId" />
+                </div>
+              </div>
+              <div class="form-group col-md-6">
+                <label for="paymentMethod">Payment Method:</label>
+                <select
+                  class="form-control"
+                  id="paymentMethod"
+                  required
+                ></select>
+              </div>
             </div>
-            <button id="createCheckout" class="btn btn-lg btn-primary btn-block" type="submit">Create checkout</button>
+            <button
+              id="createCheckout"
+              class="btn btn-lg btn-primary btn-block"
+              type="submit"
+            >
+              Create checkout
+            </button>
           </form>
         </div>
       </section>
 
       <div class="album py-5 bg-light">
         <div class="container">
-
           <script type="module">
-            import { Enabler } from '/src/main.ts';
-      
-            document.getElementById('createCheckout').addEventListener('click', async (event) => {
-              event.preventDefault();
-              const cartId = document.getElementById('cartId').value;
-              const sessionId = await getSessionId(cartId);
+            import { Enabler } from "/src/main.ts";
 
-              const enabler = new Enabler({
-                processorUrl: __VITE_PROCESSOR_URL__,
-                sessionId: sessionId,
-                currency: "EUR",
-                onComplete: ({ isSuccess, paymentReference }) => {
-                  console.log('onComplete', { isSuccess, paymentReference });
+            document.addEventListener("DOMContentLoaded", async () => {
+              const paymentMethodSelect =
+                document.getElementById("paymentMethod");
+              const response = await fetch("http://localhost:9000/jwt/token", {
```

Comment:
in order to have this working, we need a docker image running the jwt service, right? shouldn't we add in the readme or a comment in this page that points to a readme, the instructions to get that jwt service up and running?